### PR TITLE
ASC-1172 Migrate Fixtures and Helpers

### PR DESCRIPTION
**Pull request by ryan-rs**
Migrate fixtures and helpers from the 'molecule-rpc-openstack-post-deploy' repo
to this plug-in.

### CODE REVIEW

**Comment by ryan-rs** [comment]
Do not merge until staging passes for "newton", "pike" and "queens".

File: setup.py [hunk 1]
```diff
@@ -22,7 +27,7 @@
 
 setup(
     name='pytest-rpc',
-    version='0.13.0',
+    version='1.0.0-dev0',
```
Comment (zreichert) [inline setup.py]: Do you want to increment the version in this PR?

**Comment by zreichert** [comment]
@ryan-rs flake8 is failing

File: setup.py (hunk 1 above)
Comment (zreichert) [reply setup.py]: Never mind I see that it's in a separate commit.

Comment (ryan-rs) [reply setup.py]: BTW, I'm going to keep that bump in the commit history so people know that we had a dev release of the software.

**Pull request merged**